WordPress/openverse-frontend
# Add pink border to VSearchButton on group hover

## Fixes


Fixes #1284 by @obulat

## Description


This PR adds a pink border on `VSearchBar` group hover so that the normal gray border doesn't make the button look small.
It also:
- converts `VSearchBar` to TS
- removes the duplicated `VSearchBar` story
- adds visual regression testing snapshots for the header (following the conversation in #1275)
- adds some missing tapes, and also adds the `method` to the end of the tape names if it's not `GET`.

## Testing Instructions

Hover over the search bar in the header. The search button should always look like it has the same height as the search bar.

## Checklist


[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
